src/perso.ml

```diff
@@ -1961,6 +1961,14 @@ and eval_simple_str_var conf base env (_, p_auth) =
       match get_env "count" env with
       [ Vcnt c -> string_of_int c.val
       | _ -> "" ]
+  | "count1" ->
```

Comment:
I dont understand. We need more than one counter, hence count, count1 and count2!